docs/source/share_kedro_viz.md

```diff
@@ -23,11 +23,12 @@ When prompted for a project name, you can enter anything, but we will assume `Sp
 
 When your project is ready, navigate to the root directory of the project.
 
-## Update and install the dependencies
+## Publish and share Kedro-Viz automatically
 
-Kedro-Viz requires specific minimum versions of `fsspec[s3]`, and `kedro` to publish your project.
+You can automate the process of publishing and sharing your Kedro-Viz. This section describes the steps for AWS, which is the only cloud provider supported for automation at present. Integration with other cloud providers, namely Azure and GCP, will be added soon. In the absence of automated publish and share for other platforms, there is a [manual, platform-agnostic publish and share process](#platform-agnostic-sharing-with-kedro-viz) described below. You can use the manual process for sharing on static website hosts like GitHub pages, and cloud providers like Azure and GCP.
 
-You can ensure you have these correct versions by updating the `requirements.txt` file in the `src` folder of the Kedro project to the following:
+### Update and install the dependencies
+Kedro-Viz requires specific minimum versions of `fsspec[s3]`, and `kedro` to publish your project. Ensure you have these correct versions by updating the `requirements.txt` file in the `src` folder of the Kedro project to the following:
```

Comment:
"In the `src` folder"

The `requirements.txt` file doesn't live there anymore, better to just say "the `requirements.txt` file"